Tests/Veiw/PropertySpaceTest.java

```diff
@@ -0,0 +1,42 @@
+package Veiw;
+
+import View.PropertySpace;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.io.IOException;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.WindowConstants;
+
+/**********************************************************************
+ * This is a demo class for the group. It is not intended to be a concrete
+ * test class.
+ *********************************************************************/
+public class PropertySpaceTest {
+
+  public static void main(String[] args) throws IOException {
+    JFrame frame = new JFrame("Tile Test");
+    frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+    frame.setSize(250,450);
+
+    PropertySpace space = new PropertySpace(Color.CYAN,"Parrot Town", 500);
+
+    JPanel buttonPanel = new JPanel();
+    JButton inc = new JButton("Inc");
+    JButton dec = new JButton("Dec");
+    JButton quit = new JButton("Quit");
+
+    inc.addActionListener(e ->{ space.setHouses(space.getHouses() + 1);});
```

Comment:
I think that part of out code style and guidelines will mark this variable off. It should provably be written as event and not e.

Reply:
This class is a demo class that that wasn't really designed to be formal or follow naming conventions. It was only there to show everyone a design choice.  I planned on removing it as soon as this pr was approved.

Reply:
Sounds good.